build-scripts/bundle.cjs

```diff
@@ -1,7 +1,37 @@
 const path = require("path");
+const browserslist = require("browserslist");
+const lightningcss = require("lightningcss");
 const env = require("./env.cjs");
 const paths = require("./paths.cjs");
 
+const decoder = new TextDecoder();
```

Comment:
I haven't worked with encoders much.  Is there performance being gained by having this global, or are we locking up memory unnecessarily here?

Reply:
This is pretty standard formatting, and would probably improve performance.